Content/Items/Gear/Enums.cs

```diff
@@ -1,30 +1,32 @@
 ﻿namespace FunnyExperience.Content.Items.Gear
 {
 	[Flags]
-	public enum GearType : int
+	public enum GearType : long
 	{
-		Sword,
-		Spear,
-		Bow,
-		Gun,
-		Staff,
-		Tome,
-		Helmet,
-		Chestplate,
-		Leggings,
-		Ring,
-		Charm
+		None = 0,
+		Sword = 1 << 0,
+		Spear = 1 << 1,
+		Bow = 1 << 2, 
+		Gun = 1 << 3, 
+		Staff = 1 << 4,
+		Tome = 1 << 5, 
+		Helmet = 1 << 6, 
+		Chestplate = 1 << 7,
+		Leggings = 1 << 8,
+		Ring = 1 << 9,
+		Charm = 1 << 10
 	}
 
-	public enum GearRarity : int
+
+	public enum GearRarity

```

Comment:
Bitshifting should always be done for flag enums. It's generally good practice and has benefits to searching for values.

Adding none to flag enums is also a good practice.

Updating to long allows for more options, should they ever be needed.